server/utils/workflow-util.js

```diff
@@ -116,12 +116,12 @@ exports.notifyJobExecutionStatus = async ({ jobId, clusterId, WUstate, wuURL, me
         } else if (job && job.metaData.notificationSettings?.notify) {
           let cluster = await Cluster.findOne({ where: { id: clusterId } });
           let notify = job.metaData.notificationSettings.notify;
-          if (notify === 'Always') {
+          if (notify === 'Always' && WUstate !== 'not submitted' ) {
```

Comment:
Is this an HPCC status or something that is being set by Tombolo internally. Would like to understand more about this scenario

Reply:
This is something that is set by tombolo internally - If the job is not submitted because the cluster is not reachable, the pw to the cluster is incorrect, the work unit not available etc.